src/SMU-KORAD_KD3005P/main.py

```diff
@@ -150,7 +134,32 @@ def measure(self):
 
     def call(self):
         return [self.v, self.i]
-        
 
-        
-        
\ No newline at end of file
+    # wrapped commands start here
+
+    def get_identification(self):
+
+        self.port.write("*IDN?")
+        time.sleep(0.1)
+        # When using Virtual COM port, a \x00 character is at the end of the message that cannot be printed
+        identifier = self.port.read(self.port.in_waiting()).replace("\x00", "")
+        return identifier
+
+    def get_voltage_limit(self):
+        self.port.write("VSET1?")
+        vset = float(self.port.read(5))
+        return vset
+
+    def get_current_limit(self):
+        self.port.write("ISET1?")
+        iset = float(self.port.read(6)[:-1])
+        return iset
+
+    def set_output(self, state):
+
+        if not state:
+            state = 0
+        else:
+            state = 1
```

Comment:
This function could use a docstring.
set_output can be misunderstood as applying a certain voltage to the output. Also what is state? Bool or int? Probably both would work, but that just adds confusion on how it is intended to use this function